vault/values.yaml

```diff
@@ -0,0 +1,31 @@
+vault:
+  injector:
+    enabled: false
+  server:
+    serviceAccount:
+      createSecret: true
+    ha:
+      enabled: false
+      raft:
+        enabled: false
+
+vaultInit:
+  serviceAccount:
+    name: "auth-vault-init"
```

Comment:
I think that has to be a default value, hidden from values.yaml. Because I don't think there'll be much need for renaming the service account